promgen/tests/test_prometheus.py

```diff
@@ -0,0 +1,273 @@
+from unittest import mock
+
+from promgen.tests import PromgenTest
+from promgen import models
+from promgen import prometheus
+
+
+class PrometheusConfigTest(PromgenTest):
+    @mock.patch('django.dispatch.dispatcher.Signal.send')
+    def setUp(self, mock_signal):
+        user_id = 999
+        user = self.add_force_login(id=user_id, username='Tester')
+        self.__create_services([
```

Comment:
Instead of having all this structure in code, we should really be using fixtures for the tests. (Older, existing tests should also be migrated to fixtures  but that's something I should clean up)

https://docs.djangoproject.com/en/4.0/howto/initial-data/

Reply:
OK, I'll use fixtures.

Reply:
Done